### PR TITLE
Footer html & css

### PR DESCRIPTION
**Pull request by JRybelis**
- Created columns for footer content.

- Added an .underline-link style to home-page.css for links with orange underline animation effect. Attached the style to the appropriate link items in the footer columns.

- Styled 2 img button links for app and play store, positioning them in the middle vertically, using an empty <div> as space-holder. 

- Updated responsive design for the footer section.

### CODE REVIEW

**Review by martynas3113 — CHANGES_REQUESTED** [review]
Sukelti visus `<style>` tagus is HTML i css

**Review by martynas3113 — DISMISSED** [review]
LGTM

File: css/components/footer.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.container.footer {
+    background-color: var(--background-colour);
```
Comment (bemze) [inline css/components/footer.css]: variables: 
--bg-colour:

**Review by bemze — COMMENTED** [review]
background color netinkama

**Review by martynas3113 — APPROVED** [review]
LGTM

**Review by bemze — APPROVED** [review]
LGTM

**Review by pigma15 — APPROVED** [review]
lgtm. love this bit: /* reverses where transition ends, after hover */

**Pull request merged**